storage/mysql/queue.go

```diff
@@ -103,14 +104,21 @@ func (t *logTreeTX) UpdateSequencedLeaves(ctx context.Context, leaves []*trillia
 			glog.Warningf("Failed to update sequenced leaves: %s", err)
 			return err
 		}
+
+		qe, ok := t.dequeued[string(leaf.LeafIdentityHash)]
+		if !ok {
+			return fmt.Errorf("attempting to assign unknown merkleleafhash %x", leaf.MerkleLeafHash)
```

Comment:
This error message could be made clearer. How about something like "attempting to update leaf that wasn't dequeued?" + same in batched version.

Reply:
Updated